### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by ChanAndKlee**
This feature is for adding an autocomplete into the search box. 🐰

### CODE REVIEW

**Review by ChanAndKlee — COMMENTED** [review]
Please check this function

**Review by ChanAndKlee — COMMENTED** [review]
Check here

File: lab-exercise-1/AriyaFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class AriyaFeature {
+    public void feature() {
+        // this is a new feature
```
Comment (Vvidsky) [inline lab-exercise-1/AriyaFeature.java:3]: This content is not well-written. Please improve it.

File: lab-exercise-1/AriyaFeature.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 public class AriyaFeature {
     public void feature() {
         // this is a new feature
+        int x = 0;
+        x = x + 1;
```
Comment (Vvidsky) [inline lab-exercise-1/AriyaFeature.java:5]: The feature is now completed, good job!

File: lab-exercise-1/AriyaFeature.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Pontuspts) [reply lab-exercise-1/AriyaFeature.java:3]: This content is not well-written. Please improve it.

File: lab-exercise-1/AriyaFeature.java (hunk 2 above)
Comment (Pontuspts) [reply lab-exercise-1/AriyaFeature.java:5]: The feature is now completed, good job!

**Review by NoodlesCupCake — COMMENTED** [review]
Very good

File: lab-exercise-1/AriyaFeature.java (hunk 2 above)
Comment (NoodlesCupCake) [reply lab-exercise-1/AriyaFeature.java:5]: good

**Review by NoodlesCupCake — COMMENTED** [review]
Very good

File: lab-exercise-1/AriyaFeature.java (hunk 2 above)
Comment (Vvidsky) [inline lab-exercise-1/AriyaFeature.java:5]: Good job!

**Review by Vvidsky — APPROVED** [review]

**Pull request merged**